### PR TITLE
PROD-5756 Create api documentation for oab.

### PR DESCRIPTION
**Pull request by Jerjomins**
https://kaskoltd.atlassian.net/browse/PROD-5756

### CODE REVIEW

File: source/products/oab/household/rest-api.rst [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    "bicycle_module",      "string", "In:theft,kasko", "kasko"
+    "member_id",           "string", "Member id.", "1234az"
+    "street",              "string", "Street", "aj street"
+    "city",                "string", "City.", "riga"
+    "house_number",        "string", "House number", "111z"
+    "previous_insurance",  "bool", "Previous insurance of user available?", "true"
+    "account_holder_name", "string", "Account holder name.", "AJ Jerjomins"
+    "iban",                "bool",   "Iban number", "XXXXXXXXXXXXXXXXXXX"
+    "bank_name",           "string", "Bank name.", "Grizl bank"
+
```
[the remark this answers is not in the document]
Comment (Jerjomins) [reply source/products/oab/household/rest-api.rst:133]: @BearSteve wdym? its required. like all other, in policy request, where is required if, i write in the description, all others is required.

**Comment by Jerjomins** [comment]
@BearSteve thanks for feedback, than we are waiting for Andy?

**Comment by BearSteve** [comment]
Yes, Andy will contact the client

**Comment by BerSteve** [comment]
I will put `WIP` label for now.

**Comment by Jerjomins** [comment]
ok @BerSteve but i think need sometimes push Andy.

**Comment by Jerjomins** [comment]
Andy: Arturs Jerjomins ​we can leave it as it is. Issue is fine for them. Any more changes needed for this doc

**Pull request merged**